spring-boot/admin/src/main/resources/templates/admin/dashboard.html

```diff
@@ -34,7 +34,7 @@
                     });
                 }
                 function loadStatisticsPerApiConsumerInDescendingOrderByNoOfCalls(dateRangeUrlSuffix) {
-                    var url = "/admin/dashboard/statisticsPerApiConsumerInDescendingOrderByNoOfCalls.json" + dateRangeUrlSuffix;
+                    var url = "/admin/dashboard/statisticsPerApiConsumerInDescendingOrderByNoOfCalls" + dateRangeUrlSuffix;
```

Comment:
To be double-checked: is this change to be done analogously in other places as well?

Reply:
Done. I indeed found and fixed a bug at the dashboard which prevented that the API calls are updated when the time period changes.